### DISCUSSION TITLE
Type safe?

### DISCUSSION THREAD

**Discussion by TheCommCraft** [TimMcCool/scratchattach#355] [Ideas]
Will you make scratchattach type safe?

**Comment by TheCommCraft** [comment]
similar to #254

**Comment by TimMcCool** [comment]
yes, at some point in the future
you're free to help adding type hints by opening a PR

**Comment by faretek1** [comment]
i might try to add some type hints rn

**Comment by faretek1** [comment]
i added a bunch to session.py in #294 i think
i havent verified them properly though

**Comment by snoopythe3-from-scratch** [comment]
what is type safe???
